.NET/Microsoft.Recognizers.Definitions.Common/Korean/NumbersDefinitions.cs

```diff
@@ -1,243 +1,247 @@
-      public static readonly string OneNumberRangeMoreRegex3 = $@"({RangePrefixMoreRegex})\s*(?<number1>(((?!(((?!\d+))|((,)(?!\d+))|。)).)+))";
-      public static readonly string OneNumberRangeLessRegex1 = $@"((?<number2>((?!(((?!\d+))|((,)(?!\d+))|。)).)+)\s*({LessOrEqual}|{LessRegex}))|((?<number2>((?!(((?!\d+))|((,)(?!\d+))|。)).)+)\s*(이|보다|거나|또는)+\s*(그|그보다)?\s*{LessRegex}(은|다)?)|((?<number2>([영령공일이두삼사오육칠팔구]+|[십백천만억조경열]+)+)\s*(또는)?\s*(그|그보다)?\s*(미만|적게|밑))";
-      public static readonly string OneNumberRangeLessRegex2 = $@"((?<number1>((?!((\s(?!\d+))|(,(?!\d+))|。)).)+)\s*(과|에)+\s*{EqualRegex})(거나)\s*(최소)\s*(\d+)";
-      public static readonly string OneNumberRangeLessRegex3 = $@"(?:({RangePrefixLessRegex}))\s*(?<number1>(((?!((\s(?!\d+))|((,)(?!\d+))|。)).)+))";
-      public static readonly string TwoNumberRangeRegex1 = $@"(?<number1>((?!((，(?!\d+))|(,(?!\d+))|。)).)+)(위?)\s*(과|와|{TillRegex})\s*(?<number2>((?!((，(?!\d+))|(,(?!\d+))|。)).)+)(위?)\s*(사이)";
-      public static readonly string TwoNumberRangeRegex2 = $@"({OneNumberRangeMoreRegex1}|{OneNumberRangeMoreRegex2})\s*(과|또는|，|、|,)?\s*({OneNumberRangeLessRegex1})";
-      public static readonly string TwoNumberRangeRegex3 = $@"({OneNumberRangeLessRegex1})\s*(과|또는|，|、|,)?\s*({OneNumberRangeMoreRegex1}|{OneNumberRangeMoreRegex2})";
-      public static readonly string TwoNumberRangeRegex4 = $@"(?<number1>((?!((，(?!\d+))|(,(?!\d+))|。)).)+)\s*{TillRegex}\s*(?<number2>((?!((，(?!\d+))|(,(?!\d+))|。)).)+)(까지)?";
-      public static readonly string TwoNumberRangeRegex5 = $@"(?<number1>([십백천만억조경열]|(영|령|공|일|이(?!다)|두|삼|사|오|육|칠|팔|구))+)\s*{TillRegex}\s*(?<number2>([십백천만억조경열]|(영|령|공|일|이(?!다)|두|삼|사|오|육|칠|팔|구))+)\s*([십백천만억조경열]|((미만|적|낮|작|더적|더낮|더적|이하이다|까지|아래))+)";
-      public static readonly Dictionary<string, string> RelativeReferenceOffsetMap = new Dictionary<string, string>
-        {
-            { @"마지막", @"0" },
-            { @"다음", @"1" },
-            { @"뒤에서 세번째", @"-2" },
-            { @"마지막에서 두번째", @"-1" },
-            { @"마지막의 옆", @"-1" },
-            { @"마지막에서 바로 전의 것", @"-1" },
-            { @"이전 것", @"-1" },
-            { @"다음 것", @"1" },
-            { @"마지막에서 바로 전", @"-1" },
-            { @"지금의 것", @"0" },
-            { @"현재", @"0" }
-        };
-      public static readonly Dictionary<string, string> RelativeReferenceRelativeToMap = new Dictionary<string, string>
-        {
-            { @"마지막", @"end" },
-            { @"다음", @"current" },
-            { @"뒤에서 세번째", @"end" },
-            { @"마지막에서 두번째", @"end" },
-            { @"마지막의 옆", @"end" },
-            { @"마지막에서 바로 전의 것", @"end" },
-            { @"이전 것", @"current" },
-            { @"다음 것", @"current" },
-            { @"마지막에서 바로 전", @"end" },
-            { @"지금의 것", @"current" },
-            { @"현재", @"current" }
-        };
-    }
+//------------------------------------------------------------------------------
```

Comment:
Please be careful to not change the line endings of a file unless necessary. Can you revert this change and only push the conflict resolution?

Reply:
Sure, it was auto-formatted. I'll take care for future PR/Conflicts.

Reply:
I'm still seeing the line end changes after the last update.

Reply:
Where is that exactly? My be I'm referring something else.

Reply:
The whole NumbersDefinitions.cs file included in this PR after the merge conflict was resolved shows as having the line endings changed when compared to the one on /master.
On the web UI you can see it in "Files changed". NumbersDefinitons.cs shows as having  1,243 lines deleted and re-added.